lib/cli/parse.rs

```diff
@@ -1,22 +1,26 @@
 //! This module contains structs and helpers which are used by multiple subcommands related to
 //! creating deploys.
 
-use std::{convert::TryInto, fs, io, path::Path, str::FromStr};
-
-use rand::Rng;
-use serde::{self, Deserialize};
-
-use casper_types::{
-    account::AccountHash, bytesrepr, crypto, AsymmetricType, BlockHash, CLValue, DeployHash,
-    Digest, ExecutableDeployItem, HashAddr, Key, NamedArg, PublicKey, RuntimeArgs, SecretKey,
-    TimeDiff, Timestamp, UIntParseError, URef, U512,
-};
-
 use super::{simple_args, CliError, PaymentStrParams, SessionStrParams};
+#[cfg(not(any(feature = "sdk")))]
+use crate::OutputKind;
 use crate::{
-    AccountIdentifier, BlockIdentifier, GlobalStateIdentifier, JsonRpcId, OutputKind,
-    PurseIdentifier, Verbosity,
+    AccountIdentifier, BlockIdentifier, GlobalStateIdentifier, JsonRpcId, PurseIdentifier,
+    Verbosity,
 };
+#[cfg(not(any(feature = "sdk")))]
+use casper_types::SecretKey;
+use casper_types::{
+    account::AccountHash,
+    bytesrepr::{self, Bytes},
+    crypto, AsymmetricType, BlockHash, CLValue, DeployHash, Digest, ExecutableDeployItem, HashAddr,
+    Key, NamedArg, PublicKey, RuntimeArgs, TimeDiff, Timestamp, UIntParseError, URef, U512,
+};
+use rand::Rng;
+use serde::{self, Deserialize};
+#[cfg(not(any(feature = "sdk")))]
+use std::path::Path;
+use std::{convert::TryInto, fs, io, str::FromStr};
```

Comment:
I would prefer that imports stay in the order of 

```
std

external crates

crates originating from Casper

internal crate imports
```

Reply:
My VsCode is reorganizing imports on save, I am not sure about the rule applied, I guess it's alphabetic. I can try to manually re assort them.